src/core_write.cpp

```diff
@@ -207,6 +207,14 @@ void TxToUniv(const CTransaction& tx, const uint256& block_hash, UniValue& entry
                 txinwitness.push_back(HexStr(item));
             }
             in.pushKV("txinwitness", std::move(txinwitness));
+            CPubKey pubkey(tx.vin[i].scriptWitness.stack.back());
+            if (!pubkey.IsFullyValid()){
+                CScript witnessScript(tx.vin[i].scriptWitness.stack.back().begin(), tx.vin[i].scriptWitness.stack.back().end());
+                UniValue witScript(UniValue::VOBJ);
+                ScriptToUniv(witnessScript, witScript, false, true);
+                in.pushKV("witScript", std::move(witScript));
```

Comment:
I think it might make more sense to name it "witnessScript" instead of "witScript", It might be confusing to some if it is abbreviated
```suggestion
                in.pushKV("witnessScript", std::move(witScript));
```

Reply:
I agree, it needs to be changed in one more place apart from this line but yes I will apply it.
Thanks!